app/views/contexts/view_submitted_response_context.py

```diff
@@ -20,46 +20,49 @@ def build_view_submitted_response_context(
     survey_type: str,
 ) -> dict[str, Union[str, datetime, dict]]:
 
-    view_submitted_response_expired = has_view_submitted_response_expired(
-        questionnaire_store.submitted_at
-    )
-
-    if survey_type == "social":
-        submitted_text = lazy_gettext("Answers submitted.")
-    elif trad_as := questionnaire_store.metadata.get("trad_as"):
-        submitted_text = lazy_gettext(
-            "Answers submitted for <span>{ru_name}</span> ({trad_as})"
-        ).format(ru_name=questionnaire_store.metadata["ru_name"], trad_as=trad_as)
-    else:
-        submitted_text = lazy_gettext(
-            "Answers submitted for <span>{ru_name}</span>"
-        ).format(ru_name=questionnaire_store.metadata["ru_name"])
+    if questionnaire_store.submitted_at:
```

Comment:
Instead of this indentation, we should escape early. This shouldn't happen on the basis that the dev has set it up correctly, as far as typing goes, it is correctly highlight that we need to handle it, so I would just raise an exception therefore will need a test to prevent line cov failing.

Reply:
Escaping early with an exception, nesting removed, test added to give 100% cover, thanks!